riscv/csrs.cc

```diff
@@ -320,8 +320,8 @@ reg_t base_status_csr_t::compute_sstatus_write_mask() const noexcept {
   // If a configuration has FS bits, they will always be accessible no
   // matter the state of misa.
   const bool has_fs = proc->extension_enabled('S') || proc->extension_enabled('F')
-              || proc->extension_enabled_const('V');
-  const bool has_vs = proc->extension_enabled_const('V');
+              || proc->extension_enabled('V');
+  const bool has_vs = proc->extension_enabled('V');
```

Comment:
Can you please split this change into its own commit? The commit message should explain that this code only runs at reset, so never really depended on `misa.V` being constant.

Reply:
At least since ca0850358e08397dcf712a42bc6631d283646e2b this has not depended on `misa.V` being constant.